From: Daniel Echeverri <epsilon@debian.org>
Date: Sun, 29 Jul 2023 21:27:51 -0500
Subject: Don't use pip to build because this requires internet connection.
Forwarded: not-needed

--- a/Makefile
+++ b/Makefile
@@ -94,7 +94,7 @@
 	@sed -i -e 's|{{ LOGIN_DESTOP_PATH }}|"$(LOGIN_DESTOP_PATH)"|g' guake/paths.py
 	@sed -i -e 's|{{ AUTOSTART_FOLDER }}|"$(AUTOSTART_FOLDER)"|g' guake/paths.py
 
-	@$(PYTHON_INTERPRETER) -m pip install . --root "$(DESTDIR)" --prefix="/usr" || echo -e "\033[31;1msetup.py install failed, you may need to run \"sudo git config --global --add safe.directory '*'\"\033[0m"
+	@$(PYTHON_INTERPRETER) setup.py install --root "$(DESTDIR)" --prefix="/usr" --optimize=1 || echo -e "\033[31;1msetup.py install failed, you may need to run \"sudo git config --global --add safe.directory '*'\"\033[0m"
 
 	@rm -f guake/paths.py
 	@if [ -f guake/paths.py.dev ]; then mv guake/paths.py.dev guake/paths.py; fi
