--- guake-0.4.0/data/guake-prefs.desktop.in.old	2009-11-01 02:57:58.000000000 +0100
+++ guake-0.4.0/data/guake-prefs.desktop.in	2009-11-01 02:58:12.000000000 +0100
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Guake Preferences
 Name[pt_BR]=Preferências do Guake
 Comment=Comment=Configure your Guake sessions
--- guake-0.4.0/data/guake.desktop.in.old	2009-11-01 02:57:50.000000000 +0100
+++ guake-0.4.0/data/guake.desktop.in	2009-11-01 02:58:06.000000000 +0100
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Guake Terminal
 Name[pt]=Guake Terminal
 Name[pt_BR]=Guake Terminal
